{
  "id": "CVE-2025-49619",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2025-06-07T14:15:21.573",
  "lastModified": "2025-06-09T16:15:46.937",
  "vulnStatus": "Awaiting Analysis",
  "descriptions": [
    {
      "lang": "en",
      "value": "Skyvern through 0.1.85 has a Jinja runtime leak in sdk/workflow/models/block.py."
    },
    {
      "lang": "es",
      "value": "Skyvern hasta la versión 0.1.85 tiene una fuga de tiempo de ejecución de Jinja en sdk/workflow/models/block.py."
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "cve@mitre.org",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:C/C:H/I:L/A:N",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "LOW",
          "userInteraction": "NONE",
          "scope": "CHANGED",
          "confidentialityImpact": "HIGH",
          "integrityImpact": "LOW",
          "availabilityImpact": "NONE",
          "baseScore": 8.5,
          "baseSeverity": "HIGH"
        },
        "exploitabilityScore": 3.1,
        "impactScore": 4.7
      }
    ]
  },
  "weaknesses": [
    {
      "source": "cve@mitre.org",
      "type": "Secondary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-1336"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://cristibtz.github.io/posts/CVE-2025-49619/",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://github.com/Skyvern-AI/skyvern/commit/db856cd8433a204c8b45979c70a4da1e119d949d",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://cristibtz.blog/posts/CVE-2025-49619/",
      "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0"
    }
  ]
}